index.js

```diff
@@ -165,6 +165,26 @@ class Conf {
 		this.events.emit('change');
 	}
 
+	createDir() {
+		// Ensure the directory exists as it could have been deleted in the meantime
+		makeDir.sync(path.dirname(this.path));
+	}
+
+	watch() {
+		this.createDir();
+
+		let wait = false;
+		fs.watch(path.dirname(this.path), {encoding: this.encryptionKey ? null : 'utf8'}, () => {
```

Comment:
`fs.watch` accepts a filename, so no need to get the directory.

The `encoding` option setting here is incorrect.